### PR TITLE
fix(curriculum): python for everybody

### LINKED ISSUE #38925: Python for Everybody - Introduction: Elements of PythonPassed
**Describe your problem and how to reproduce it:**
All the code should be aligned at the same starting position.

**Add a Link to the page with the problem:**
https://www.freecodecamp.dev/learn/scientific-computing-with-python/python-for-everybody/introduction-elements-of-python

**Tell us about your browser and operating system:**
* Browser Name: Google Chrome
* Browser Version: 78.0.3904.87 (Build oficial) (64 bits)
* Operating System: Windows 10

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
image

### LINKED ISSUE #38927: Python for Everybody - Python Functions
**Describe your problem and how to reproduce it:**
Not major, but seeing as there are not any list labels such as
```
a)
b)
c)
...
```
It is confusing answering this:
image

**Add a Link to the page with the problem:**
https://www.freecodecamp.dev/learn/scientific-computing-with-python/python-for-everybody/python-functions

### LINKED ISSUE #38930: Python for Everybody - Networking: Write a Web Browser
**Describe your problem and how to reproduce it:**
Part of the code is contained within the parent `pre` tag, whilst the rest is contained within the child `p` tag. This leads to different font-sizes as well as different font-family. Also, causes separation/apparent indentation.

**Add a Link to the page with the problem:**
https://www.freecodecamp.dev/learn/scientific-computing-with-python/python-for-everybody/networking-write-a-web-browser

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
image

### LINKED ISSUE #38931: Python for Everybody - Objects: A Sample Class
**Describe your problem and how to reproduce it:**
Code formatting needs fix, and solution is not correct. Solution should be:
>2
>4

But this is not even an option.

**Add a Link to the page with the problem:**
https://www.freecodecamp.dev/learn/scientific-computing-with-python/python-for-everybody/objects-a-sample-class

### LINKED ISSUE #38939: Python for Everybody: malformed sentence in "Variables, Expressions, and Statements" challenge
Challenge: https://www.freecodecamp.dev/learn/scientific-computing-with-python/python-for-everybody/variables-expressions-and-statements

Sentence has an extra "is" in it.

What is the symbol is used in an assignment statement?

Should be:

What is the symbol used in an assignment statement?

### LINKED ISSUE #38941: Python for Everybody: python functions, better indication of multiple choice answers
Challenge: https://www.freecodecamp.dev/learn/scientific-computing-with-python/python-for-everybody/python-functions

I'd prefer "Answers b and c are both true" instead of "b and c are both true", just to avoid any confusion. Or some other clear indication of multiple choice answers.

### LINKED ISSUE #38964: Spelling issue - scientific-computing-with-python/python-for-everybody/networking-protocol
**Describe your problem and how to reproduce it:**
Spelling issue at scientific-computing-with-python/python-for-everybody/networking-protocol
Current text: What type of HTTP **is** request is usually used to access a website?
Should be: What type of HTTP request is usually used to access a website?
Screen Shot 2020-05-30 at 9 27 55 AM

**Add a Link to the page with the problem:**
https://www.freecodecamp.dev/learn/scientific-computing-with-python/python-for-everybody/networking-protocol

**Tell us about your browser and operating system:**
* Browser Name: Brave
* Browser Version: Version 1.9.72 Chromium: 81.0.4044.138 (Official Build) (64-bit)
* Operating System: macOS Catalina version 10.15.4

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #38925 
Closes #38939 
Closes #38941 
Closes #38927 
Closes #38930 
Closes #38931 
Closes #38964

### CODE REVIEW

**Comment by ojeytonwilliams** [comment]
Hey @Sky020 what do you think about using markdown instead?  I'd rather we use it as much as possible, so when people look at the curriculum for inspiration they see how it can be done.

I appreciate that switching is a chore, so I'd be happy to make the changes myself.

**Comment by Sky020** [comment]
@ojeytonwilliams , sure. Shall i refactor the questions as well?

**Comment by ojeytonwilliams** [comment]
@Sky020 wherever possible, please.  It should make things easier to read.  However, if there are things that are hard to express in Markdown, don't worry about them.

**Comment by Sky020** [comment]
There is conflict with these and #38949, #38951. This should be prioritised.

File: curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/comparing-and-sorting-tuples.english.md [hunk 1]
```diff
@@ -18,13 +18,25 @@ More resources:
...
+    lst = sorted(lst, reverse=True)
+    print(lst)
+    ```
   answers:
-    - 'print( sorted( [ (k,v) for k,v in counts.items() ] ) )'
-    - 'print( [ (k,v) for k,v in counts.items().sorted() ] )'
-    - 'print( sorted( [ (k,v) for k,v in counts.keys() ] ) )'
-    - 'print( [ (k,v) for k,v in counts.values().sort() ] )'
+    - |
+        print( sorted( [ (k,v) for k,v in counts.items() ] ) )
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/comparing-and-sorting-tuples.english.md:33]:
```suggestion
        print( sorted( [ (v,k) for k,v in counts.items() ] ) )
```
The code in the question flips the tuple, so the answer has to as well. Also, it should look better in a ```py block, see https://github.com/freeCodeCamp/freeCodeCamp/issues/38942

File: curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/comparing-and-sorting-tuples.english.md [hunk 2]
```diff
@@ -18,13 +18,25 @@ More resources:
...
+    - |
+        print( [ (k,v) for k,v in counts.items().sorted() ] )
+    - |
+        print( sorted( [ (k,v) for k,v in counts.keys() ] ) )
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/comparing-and-sorting-tuples.english.md:37]:
```suggestion
        print( sorted( [ (v,k) for k,v in counts.keys() ] ) )
```
Just so it's not completely obvious which the right answer is!

File: curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/regular-expressions.english.md [hunk 3]
```diff
@@ -17,13 +17,20 @@ videoId: Yud_COr6pZo
...
+    - |
+        \S
+    - |
+        \s
+    - |
+        .
+    - |
+        \_
+    - |
+        \.
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/regular-expressions.english.md:33]:
```suggestion
        \\.
```
For some reason \\. escapes the dot (not sure why), but that means you need an extra slash.

File: curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/networking-write-a-web-browser.english.md [hunk 4]
```diff
@@ -37,10 +37,14 @@ question:
     ```
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/networking-write-a-web-browser.english.md:37]: It's slightly out of scope of this PR, but the indentation is wrong above.  Could you line everything (lines 33 through 34) up with `data`?

It should be

```py
while True:
    data = mysock.recv(512)
    if len(data) < 1:
        break
    print(data.decode(),end='')
mysock.close()
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Nice work @Sky020.  I found a few problems when I went through this, but I think they were all pre-existing issues.  Might as well fix them here, since they're all pretty minor.

**Comment by Sky020** [comment]
@ojeytonwilliams , Thank you, for catching those. I have committed those changes.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM 👍

**Review by moT01 — APPROVED** [review]
LGTM @Sky020 🎉 Thanks for contributing to freeCodeCamp 🎉

**Pull request merged**